microraiden/microraiden/channel_manager/state.py

```diff
@@ -35,22 +36,22 @@ def dict_factory(cursor, row):
 );
 -- deposit and balance have length of 78 to fit uint256
 CREATE TABLE `channels` (
-    `sender`            CHAR(42),
-    `open_block_number` INTEGER,
-    `deposit`           DECIMAL(78,0),
-    `balance`           DECIMAL(78,0),
+    `sender`            CHAR(42)        NOT NULL,
```

Comment:
A general note: if you don't want to mess around with SQL table creation and queries, check out peewee. It offers a nice abstraction layer with custom model classes. You could theoretically just swap out the backend (SQLite, MySQL, PostgreSQL) in a few lines without changing most of the Python code:

http://docs.peewee-orm.com/en/latest/peewee/quickstart.html#quickstart

Reply:
Sure we can switch to something like this, @andrevmatos suggested i.e. SQLAlchemy. 
If it makes the code more clean, I am all for it!